### DISCUSSION TITLE
Confused on what file to use to update my blueretro

### DISCUSSION THREAD

**Discussion by blakezilla** [darthcloud/BlueRetro#300] [Q&A]
Hi there, using for PS1/PS2, on the older ones I would use the BlueRetro_psx_ps2_spiffs.bin file, but in the newer releases there is no such file. What do I use?

**Comment by darthcloud** [comment]
BlueRetro_psx_external.bin

**Reply by blakezilla** [reply]
Thank you so much! One more question if you want me to create another one or it is not related, I am having a lot of issues with blue retro consistently pressing the start button in-game and doesn't allow me the use of the analogue sticks. I am using a DS5 controller and I have xstation/PS1 Digital. Have you heard of this happening?

**Reply by darthcloud** [reply]
Random press is an issue when using a PS1 digital/BlueRetro with some PS1 see #102 , I got not definitive answer yet for that.

Analog work like a real controller, If the game don't turn on analog itself you need to press the analog button, which is the PS button on DS5 with default mapping.

**Reply by blakezilla** [reply]
Thank you so much you are a godsend! Thanks so much for your work!
